src/components/Navigation/NavBar/NavItems/NavItem/NavItem.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import classes from './NavItem.module.css'
+import PropTypes from 'prop-types'
 
 const navItem = (props) => (
-    <li className={classes.NavItem} onClick={props.clicked}>
-        <div>{props.children}</div>
+    <li className={classes.NavItem} >
+        <button aria-label={props.ariaLable? props.ariaLable : null} onClick={props.clicked}>{props.children}</button>
```

Comment:
Have I mentioned "short circuiting"? Something like `props.ariaLable ? props.ariaLable : null` can be written as `props.ariaLable || null`. Give it a google, it's a handy pattern. 

Something to watch out for with this kind of code, ternary or short circuit, are all the possible falsey values that could be passed in here. For example, what would happen if someone wanted the label to be `0`? Maybe that's not likely here, but when making a straight forward check like this it's always good to think about the possible falsey values that might actually be desired. In this case, you might want to more explicitly check if `props.ariaLable === undefined
`. Note, when you need to be more specific like that then short circuiting's out and you should stick with a ternary.

Reply:
Oh, I've seen this when people set up express ports! I never knew this had a name. Now that you mention falsey data, I now see it. I'll keep this in mind when I use this in the weather app!